instrumentation/cassandra/cassandra-4.4/testing/src/main/java/io/opentelemetry/testing/cassandra/v4_4/AbstractCassandra44Test.java

```diff
@@ -0,0 +1,8 @@
+package io.opentelemetry.testing.cassandra.v4_4;
+
+import io.opentelemetry.cassandra.v4.common.AbstractCassandraTest;
+
+public abstract class AbstractCassandra44Test extends AbstractCassandraTest {
+
+  // TODO add reactive tests here
```

Comment:
can you rewrite the couple of reactive groovy tests here in Java?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6441/commits/2385797ceceaa103e0daaadceaafda5f6246ea75